docs/concepts/executions.md

```diff
@@ -1,37 +1,44 @@
+
+Each task run will produce one or more attempts. Most task runs will have only one attempt, but you can configure [retries](../developer-guide/retries) for a task. If retries have been configured, they will generate new attempts for each failed task run.
+
 
 ## Outputs 
-Each task can generate output data that can be reused in other tasks of the current execution.
+
+Each task can generate output data that other tasks of the current execution can use.
+
 
 ## Metrics 
-Each task can generate metrics data that may be useful in understanding what the current task is doing. 
-Examples of metrics you might see include: `file.size`, `row.count`, etc...
 
-There are 2 kinds of metrics: 
-* `Duration` 
-* `Counter`  
+Each task can generate metrics data that may be useful in understanding what the current task is doing. For example, `file.size`, `row.count`, etc.
+
+There are two kinds of metrics. Those can measure duration or a counter. 
 
 
 ## State 
-A Task Run or an Execution can be in a particular state. 
-
-The possible States are defined below:  
-* `CREATED`: The Task Run or Execution is not yet handled internally, waiting to be processed. This usually means the Execution is in a queue.
-* `RUNNING`: The Task or Execution is currently running.
-* `SUCCESS`: The Task or Execution has completed successfully.
-* `WARNING`: The Task Run or Execution is exhibiting unintended behaviour, but the execution continued and was flagged with a warning.
-* `FAILED`: The Task Run or Execution exhibited unintended behaviour that caused the execution to fail.
-* `KILLING`: A command was run asking the Task Run or Execution to be killed, and the system is in the process of killing the associated tasks.
-* `KILLED`: A Task Run or Execution was killed (upon request), and no more tasks will be run for that process.
-* `RESTARTED`: This status is a transitive status that essentially means `CREATED`, but for a process that has already been executed and failed, and has been restarted.
+
+An Execution or a Task Run can be in a particular state. 
+
+There are eight possible states:    
+* `CREATED`: The Execution or Task Run is waiting to be processed. This state usually means that the Execution is in a queue and has not been handled internally.
+* `RUNNING`: The Execution or Task Run is currently running.
+* `SUCCESS`: The Execution or Task Run has been completed successfully.
+* `WARNING`: The Execution or Task Run exhibited unintended behavior, but the execution continued and was flagged with a warning.
+* `FAILED`: The Execution or Task Run exhibited unintended behavior that caused the execution to fail.
+* `KILLING`: A command was issued that asked for the Execution or Task Run to be killed. The system is in the process of killing the associated flow or tasks.
+* `KILLED`: An Execution or Task Run was killed (upon request), and no more tasks will run for that process.
```

Comment:
```suggestion
* `KILLED`: An Execution or Task Run was killed (upon request), and no more tasks will run for that it.
```

I think we use nowhere the term `process` so better to avoid it